owl/SOMA-OBJ.owl

```diff
@@ -787,6 +812,36 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasReferencePose -->
+
+    <owl:DatatypeProperty rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasReferencePose">
+        <rdfs:subPropertyOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#hasShapeParameter"/>
+        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ShapeRegion"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
+        <rdfs:comment>Gives the default pose of a shape in its own coordinate system.
+
+It is sometimes the case that a shape described via a mesh or a primitive needs to be translated and/or rotated in order to correspond to the object it is a shape of. This can happen, e.g., when a robot description file places the center point of a link at a different location to the centroid of the mesh giving that link&apos;s shape.</rdfs:comment>
```

Comment:
in case of sticking to string values, I think it would be good to write what the format of poses is, e.g. `[frame,position,quaternion]`

Reply:
True. The same format should be used as for 6DoF pose region.

Reply:
looking at SOMA.owl `6DPose` uses three data properties `hasOrientationVector`, `hasTranslationVector` and `hasReferenceFrame`